websites/T/TVNOW/dist/metadata.json

```
@@ -9,7 +9,12 @@
     "nl": "TVNOW biedt spannende films en series. Ook zijn er veel shows en documentaires te vinden in de online stream! Bekijk nu uw favoriete programma online!",
     "de": "TVNOW bietet Dir spannende Filme und Serien. Auch viele Shows und Dokus sind im Online-Stream zu finden! Deine Lieblingssendung jetzt online anschauen!"
   },
-  "url": "www.tvnow.de",
+  "url": [
+    "www.tvnow.at",
+    "tvnow.at",
+    "tvnow.de",
+    "www.tvnow.de"
```

You shouldn't need the "www"
```suggestion
    "tvnow.at",
    "tvnow.de",
```